fmriprep/workflows/bold/t2s.py

```diff
@@ -67,14 +68,14 @@ def init_bold_t2s_wf(echo_times,
     """
     workflow = Workflow(name=name)
     workflow.__desc__ = """\
-A T2* map was estimated from preprocessed BOLD by fitting to a
-monoexponential signal decay model with log-linear regression.
-For each voxel, the maximal number of echoes with high signal in that voxel was
+A T2* map was estimated from the preprocessed BOLD by fitting to a monoexponential signal
+decay model with log-linear regression.
+For each voxel, the maximal number of echoes with reliable signal in that voxel were
 used to fit the model.
-The T2* map was used to optimally combine preprocessed BOLD across
-echoes following the method described in @posse_t2s and was also retained as
-the BOLD reference.
-"""
+The calculated T2* map was then used to optimally combine preprocessed BOLD across
```

Comment:
What does this mean? (this is not a question about the PR, it is to fill a gap of ignorance on my side, and potentially ask for a little amend for people of my level of knowledge about MEEPI)

Reply:
We use the T2* map to determine the weightings given to each echo in the optimal combination. Should I update the text here, to reference that ? It's explained in the reference, but I can also put more information, here !

Reply:
I think it is okay, let's not delay this PR :) Thanks!